sdk/identity/Azure.Identity/src/AzureAuthorityHosts.cs

```diff
@@ -27,6 +27,7 @@ public static class AzureAuthorityHosts
         /// <summary>
         /// The host of the Microsoft Entra authority for tenants in the Azure German Cloud.
         /// </summary>
+        [Obsolete("Microsoft Cloud Germany was closed on October 29th, 2021.")]
```

Comment:
We'll need to remove references to this from the various tests to avoid the build errors.

Reply:
Isn't `Obsolete`'s default behavior, i.e., its optional parameter remaining `false`, meant to not break existing usage?
If not, I'd happily remove them all, of course. 😉

Reply:
We build with warnings as errors, so that is what causes problems.

Reply:
I see. Removing the relevant tests will be the first thing tomorrow morning, stay tuned.

Reply:
Modified 1 test to succeed again instead of removing it & removed `AzureGermany` from an enumeration to let dependent tests succeed again as well.
Unfortunately, the PS script apparently needs to be run locally.. 🙄

Reply:
@christothes Can you guide me through this? I simply don't know how to fix that path error & would've expected that folders with spaces in their name are no big deal for today's build scripts?! 🤔😅

Reply:
Sorry for the delay - I've been out of the office and am just catching up today. If you give me rights to push to your fork, I should be able to fix this for you.

Reply:
Alright, thanks for letting me know. Take your time, though. Invitation was sent just now. 😎